site/src/components/Logs/LogLine.tsx

```diff
@@ -0,0 +1,89 @@
+import type { Interpolation, Theme } from "@emotion/react";
+import type { FC, HTMLAttributes } from "react";
+import type { LogLevel } from "api/typesGenerated";
+import { MONOSPACE_FONT_FAMILY } from "theme/constants";
+
+export const DEFAULT_LOG_LINE_SIDE_PADDING = 24;
+
+export interface Line {
+  time: string;
+  output: string;
+  level: LogLevel;
+  sourceId: string;
+}
+
+type LogLineProps = {
+  level: LogLevel;
+} & HTMLAttributes<HTMLDivElement>;
+
+export const LogLine: FC<LogLineProps> = ({ level, ...divProps }) => {
+  return (
+    <div
+      css={styles.line}
+      className={`${level} ${divProps.className} logs-line`}
+      {...divProps}
+    />
+  );
+};
+
+export const LogLinePrefix: FC<HTMLAttributes<HTMLSpanElement>> = (props) => {
+  return <span css={styles.prefix} {...props} />;
+};
+
+export const LogLineSpace: FC = () => {
+  return <span css={styles.space} />;
+};
```

Comment:
I'm not crazy about this component. I feel like there's gotta be a better way to add this padding/gap. A non-semantic empty span is a code smell to me.